packages/hawtio/src/util/objects.ts

```diff
@@ -3,6 +3,14 @@ export function isObject(value: unknown): value is object {
   return value != null && (type === 'object' || type === 'function')
 }
 
+export function cloneObject<T>(value: T): T {
+  return JSON.parse(JSON.stringify(value))
```

Comment:
at some point in the future we should replace this with `structuredClone()` builtin fn.

Reply:
Captured with issue https://github.com/hawtio/hawtio-next/issues/106